wire-ios/Wire-iOS/Sources/Analytics/AnalyticsDecryptionFailedObserver.swift

```diff
@@ -42,7 +44,7 @@ final class AnalyticsDecryptionFailedObserver: NSObject {
                 trackingInfo[$0] = value
             }
         }
-
+        logger.info("Message Cannot Be Decrypted \(trackingInfo)")
```

Comment:
Instead of logging here, I suggest to add the log where the `ZMConversation.failedToDecryptMessageNotificationName` is posted.

Reply:
updated.